manifest.json

```diff
@@ -10,17 +10,23 @@
   },
   "permissions": [
     "activeTab",
-    "storage"
+    "storage",
+    "tabs"
   ],
   "content_scripts": [
     {
       "matches": [
-        "https://openclassrooms.com/fr/dashboard/*"
+        "https://openclassrooms.com/*"
```

Comment:
I don't think running the extension on all pages of `openclassrooms.com` is necessary here.
What do you think of
```javascript
[
"https://openclassrooms.com/fr/dashboard/*", // french version
"https://openclassrooms.com/en/dashboard/*" // english version
]
```

Reply:
I agree. Solution tested and working on local.